objects/obj_player/Draw_64.gml

```diff
@@ -1,13 +1,33 @@
-/// @description draw weapon text and ammo GUI
+/// @description draw weapon text and ammo GUI and coins
 
 //weapon text
-display_set_gui_size(640,360);
+display_set_gui_size(camera_get_view_width(view_camera[0]), camera_get_view_height(view_camera[0]));
 draw_set_font(fnt_combo2);
 draw_set_halign(fa_right);
-scr_Draw_Text_Outlined(624,40,gun.name,c_white);
+scr_Draw_Text_Outlined(camera_get_view_width(view_camera[0])-16,40,gun.name,make_color_rgb(242,240,229));
 
 //draw ammo
 var ammo = gun.ammo[bullet_index];
 for(i = 0; i < gun.bullets_per_bounce; i++) {
-	draw_sprite(ammo.gui_sprite,i+gun.current_bullets<gun.bullets_per_bounce,624-(i*(sprite_get_width(ammo.gui_sprite)+4)),16);
-}
\ No newline at end of file
+	if (gun_array[current_gun] != paintball_gun) and (gun_array[current_gun] != laser_gun) {
+		draw_sprite(ammo.gui_sprite,i+gun.current_bullets<gun.bullets_per_bounce,(camera_get_view_width(view_camera[0])-16)-(i*(sprite_get_width(ammo.gui_sprite)+4)),16); 
+	}else if (gun_array[current_gun] = paintball_gun) {
+		draw_sprite(ammo.gui_sprite,((i+gun.current_bullets<gun.bullets_per_bounce)*10)+((i+gun.current_bullets>=gun.bullets_per_bounce)*i),(camera_get_view_width(view_camera[0])-16)-(i*(sprite_get_width(ammo.gui_sprite)+4)),16); 
+	}
+}
+//laser
+if (gun_array[current_gun] = laser_gun) {
+	value_ = gun.current_bullets / gun.bullets_per_bounce;
+	draw_sprite(ammo.gui_sprite,0,(camera_get_view_width(view_camera[0])-16),16);
+	draw_sprite_part(ammo.gui_sprite,1,0,0,sprite_get_width(ammo.gui_sprite)*value_,sprite_height,(camera_get_view_width(view_camera[0])-16)-sprite_get_width(ammo.gui_sprite),16-(sprite_get_height(ammo.gui_sprite)/2));
+		
+	//draw_sprite(ammo.gui_sprite,((i+gun.current_bullets<gun.bullets_per_bounce)*10)+((i+gun.current_bullets>=gun.bullets_per_bounce)*i),(camera_get_view_width(view_camera[0])-16)-(i*(sprite_get_width(ammo.gui_sprite)+4)),16); 
```

Comment:
Is there a reason for keeping this commented code? if so can you add a comment to describe its purpose?